src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs

```diff
@@ -136,10 +140,8 @@ bool ProcessOutput (string singleLine)
 					errorText.Clear ();
 				}
 				file = match.Groups ["file"].Value;
-				line = int.Parse (match.Groups ["line"].Value) + 1;
```

Comment:
@dellis1972 I'm a little confused about adding `+ 1` to the line number here, where it gets `- 1` later on:

https://github.com/xamarin/xamarin-android/blob/4a71aa6217dabc75b8616ee72440d85352868118/src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs#L98

Do you know the history behind that? It also does similar with `column`.

Is it something about BOMs?

Reply:
If I remember rightly, it was to do with how it appears in the IDE. Since we do change the code to strip out BOMs etc when we copy the files to the directory where we are going to compile thngs, its probably related to that.